core/controllers/question_editor.py

```diff
@@ -30,9 +30,9 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, comma_separated_skill_ids):
+    def post(self):
         """Handles POST requests."""
-        skill_ids = comma_separated_skill_ids.split(',')
+        skill_ids = self.payload.get('skill_ids')
```

Comment:
Should it raise any exception if the payload doesn't have skill_ids? If so, test this case as well.

Reply:
Done